cocos/core/renderer/core/pass.ts

```diff
@@ -381,6 +385,14 @@ export class Pass {
         }
     }
 
+    public getInstancedBuffer (extraKey = 0) {
```

Comment:
getInstancedBuffer(0) and getInstancedBuffer() will collide, are you sure it's ok ?

Reply:
It's the same behavior as before